src/Icons/doc/index.rst

```diff
@@ -19,10 +19,60 @@ Installation
 
     $ composer require symfony/ux-icons
 
-Icons?
+Icons
 ------
 
-No icons are provided by this package but there are several ways to include and render icons.
+The ``symfony/ux-icons`` package aspires to offer simple and easy-to-use ways to
+handle SVG icons in your Symfony application.
+
+SVG Icons
+~~~~~~~~~
+
+[SVG](https://en.wikipedia.org/wiki/SVG) (Scalable Vector Graphics) is an XML-based
+vector image format. SVGs are a great way to add scalable, resolution-independent
+graphics to your website. They can be styled with CSS, combined, animated, reused..
+for a fraction of the file size of bitmap images.
+
+Making them the perfect choice for icons.
+
+
+Icon Sets
+~~~~~~~~~
+
+There are many icon sets available, each with their own unique style and set of icons.
+
+
+Icon identifiers
+~~~~~~~~~~~~~~~~
+
+In the UX Icons component, icons are referenced using an unique identifier that
+follows one of the following syntaxes:
+
+* ``prefix``:``name``  (e.g. ``mdi:check``, ``bi:check``, ``editor:align-left``)
+* ``name`` only (e.g. ``check``, ``close``, ``menu``)
+
+Icon Name
+^^^^^^^^^
+
+The ``name`` is the... name of the icon, without the file extension (e.g. ``check``).
+
+.. caution::
+
+    The name must match a standard ``slug`` format: ``[a-z0-9-]+(-[0-9a-z])+``.
+
+Icon Prefix
```

Comment:
In this section I'm missing how to find the prefix for the icon set. On iconify website is not that obvious (you can find it in the URL when viewing some icon).